cumulusci/robotframework/perftests/short/collection_perf.robot

```diff
@@ -66,9 +66,12 @@ Insert 200 Prospecting Opportunities
 *** Test Cases ***
 
 Perftest - Insert 200 Contacts
+    Start Perf Timer
     Insert 200 Contacts
+    End Perf Timer
```

Comment:
This should probably be done in a teardown. Otherwise, if `Insert 200 Contacts` fails, the keyword will never be called. Likewise, `Start Perf Timer` should probably be in a setup.

```
Perftest - Insert 200 Contacts
    [Setup]  Start Perf Timer
    Insert 200 Contacts
    [Teardown]  End Perf Timer
```

Reply:
I will double check that there is no memory leak etc. if that happens, but otherwise I wouldn't expect a failed test to "report" its runtime.

Reply:
The "timer" is just a datetime object stored in the tests local variables, so memory leakage is not a problem. So I think I will leave the example as-is.